app/components/results-list/component.js

```diff
@@ -10,6 +10,14 @@ export default Ember.Component.extend({
         let data = this.processData(this.get('data.hits.hits'));
         this.set('data', data);
     },
+    didRender() {
```

Comment:
how does didRender differ from willRender? I'm thinking if we run this on did render, we're doing something that causes another render.